metrics-operator/controllers/common/providers/dynatrace/dynatrace.go

```diff
@@ -53,23 +53,53 @@ func (d *KeptnDynatraceProvider) EvaluateQuery(ctx context.Context, metric metri
 	d.Log.Info("Running query: " + qURL)
 	ctx, cancel := context.WithTimeout(ctx, 20*time.Second)
 	defer cancel()
-	req, err := http.NewRequestWithContext(ctx, "GET", qURL, nil)
+	result, b, err := d.performRequest(ctx, provider, qURL)
+	if err != nil {
+		return "", b, err
+	}
+	r := fmt.Sprintf("%f", d.getSingleValue(result))
+	return r, b, nil
+}
+
+func (d *KeptnDynatraceProvider) EvaluateQueryForStep(ctx context.Context, metric metricsapi.KeptnMetric, provider metricsapi.KeptnMetricsProvider) ([]string, []byte, error) {
+	if metric.Spec.Range == nil {
+		return nil, nil, fmt.Errorf("spec.range is not defined!")
+	}
+	baseURL := d.normalizeAPIURL(provider.Spec.TargetServer)
+	query := url.QueryEscape(metric.Spec.Query)
+	qURL := baseURL + "v2/metrics/query?metricSelector=" + query + "&from=now-" + metric.Spec.Range.Interval + "/" + metric.Spec.Range.Step
```

Comment:
I was just checking the DT API docs again and found that we have to provide the `step` as the `resolution` query parameter: https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/get-data-points, instead of adding the `/<step>` to the `from` parameter

Reply:
I mistook alignment as resolution, my bad. Fixed it.